packages/oauth/oauth-provider/src/token/token-claims.ts

```diff
@@ -0,0 +1,14 @@
+import { OAuthScope } from '@atproto/oauth-types'
+import { ClientId } from '../client/client-id.js'
+import { TokenId } from './token-id.js'
+
+export type TokenClaims = {
+  jti: TokenId
+  sub: string
+  iat: number
+  exp: number
+  cnf?: { jkt: string }
+  aud?: string | [string, ...string[]]
+  scope?: OAuthScope
+  client_id?: ClientId
+}
```

Comment:
Should `iss` be included here, consistent with the `requiredClaims` list in `decodeToken()`?  I assume there is a reason it's not here, but it's not jumping out at me.